jlpm develop fails due to lack of `buildutils/lib/develop.js` file

In a fresh Conda environment with `pip`, `python`, and `nodejs` installed, I have the tip of the `main` branch of Notebook checked out. After running `pip install -e ".[dev,test]"` successfully, per [the contributor docs](https://jupyter-notebook.readthedocs.io/en/stable/contributing.html), I run `jlpm develop`. This produces an error:

```
 jlpm develop
Installing /Users/jweill/git/notebook/notebook/labextension -> @jupyter-notebook/lab-extension
Removing: /opt/miniconda3/envs/notebook-20240523/share/jupyter/labextensions/@jupyter-notebook/lab-extension
Symlinking: /opt/miniconda3/envs/notebook-20240523/share/jupyter/labextensions/@jupyter-notebook/lab-extension -> /Users/jweill/git/notebook/notebook/labextension
node:internal/modules/cjs/loader:1146
  throw err;
  ^

Error: Cannot find module '/Users/jweill/git/notebook/buildutils/lib/develop.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1143:15)
    at Module._load (node:internal/modules/cjs/loader:984:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.12.2
```

There is a `buildutils` directory under my local `notebook` repository, but there is no `lib` directory under it.

https://github.com/jupyter/notebook/blob/0a66f40c468ae2bf15faceba204288f614c96969/package.json#L30

The first command in the rvalue above succeeds. The latter does not, because the directory and file do not exist.

With `jlpm` version 3.5.0 installed, I still get an error after running `jlpm run build && jlpm develop` in my notebook environment:

```
$ jlpm run build && jlpm develop
lerna notice cli v6.6.2
lerna info versioning independent

    ✔  @jupyter-notebook/application:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/buildutils:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/ui-components:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/lab-extension:build:lib  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/terminal-extension:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/docmanager-extension:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/documentsearch-extension:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/tree:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/console-extension:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/notebook-extension:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/help-extension:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/application-extension:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/tree-extension:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/app:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/lab-extension:build:labextension:dev  [local cache]
    ✔  @jupyter-notebook/lab-extension:build  [existing outputs match the cache, left as is]
    ✔  @jupyter-notebook/metapackage:build  [existing outputs match the cache, left as is]

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  Lerna (powered by Nx)   Successfully ran target build for 15 projects and 2 tasks they depend on (102ms)
 
   Nx read the output from the cache instead of running the command for 17 out of 17 tasks.
 
Installing /Users/jweill/git/notebook/notebook/labextension -> @jupyter-notebook/lab-extension
Removing: /opt/miniconda3/envs/notebook-20240523/share/jupyter/labextensions/@jupyter-notebook/lab-extension
Symlinking: /opt/miniconda3/envs/notebook-20240523/share/jupyter/labextensions/@jupyter-notebook/lab-extension -> /Users/jweill/git/notebook/notebook/labextension
node:internal/modules/cjs/loader:1146
  throw err;
  ^

Error: Cannot find module '/Users/jweill/git/notebook/buildutils/lib/develop.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1143:15)
    at Module._load (node:internal/modules/cjs/loader:984:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.12.2
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -24,6 +24,10 @@ mamba activate notebook
 # Install package in development mode
 pip install -e ".[dev,test]"
 
+# Install dependencies and build packages
+jlpm
+jlpm build
+
 # Link the notebook extension and @jupyter-notebook schemas
 jlpm develop
 

